README.md

```diff
@@ -78,37 +78,75 @@ PyMilo is an open source Python package that provides a simple, efficient, and s
 
 
 ## Usage
-### Model preparation 
+Imagine you want to train a `LinearRegression` model representing this equation: `y = 1 * x_0 + 2 * x_1 + 3`. You make your data points (`X`, `y`) and train your model as follows.
 ```pycon
->>> from sklearn import datasets
->>> from pymilo import Export, Import
+>>> import numpy as np
 >>> from sklearn.linear_model import LinearRegression
->>> import os
->>> X, Y = datasets.load_diabetes(return_X_y=True)
->>> threshold = 20
->>> X_train, X_test = X[:-threshold], X[-threshold:]
->>> Y_train, Y_test = Y[:-threshold], Y[-threshold:]
->>> model = LinearRegression()
->>> #### Train the model using the training sets
->>> model.fit(X_train, Y_train)
+>>> X = np.array([[1, 1], [1, 2], [2, 2], [2, 3]])
+>>> y = np.dot(X, np.array([1, 2])) + 3
+ # y = 1 * x_0 + 2 * x_1 + 3
+>>> model = LinearRegression().fit(X, y)
+>>> pred = model.predict(np.array([[3, 5]]))
+# pred = [16.] (=1 * 3 + 2 * 5 + 3)
 ```
-### Save model 
+
+Using PyMilo `Export` you can easily export, serialize, your trained model into a `.json` file.
```

Comment:
Using PyMilo `Export` class, you can easily serialize and export your trained model into a JSON file.